.eslintrc.json

```diff
@@ -20,9 +20,12 @@
     "ecmaVersion": "latest",
     "sourceType": "module"
   },
+
   "plugins": ["react", "@typescript-eslint"],
   "rules": {
     "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-unused-vars": "error",
+    "react/react-in-jsx-scope": "off",
```

Comment:
is it needed?

Reply:
I learned something new
https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj